app/domain/authentication/authn_jwt/fetch_token_from_credentials.rb

```diff
@@ -0,0 +1,22 @@
+module Authentication
+  module AuthnJwt
+
+    FetchTokenFromCredentials ||= CommandClass.new(
```

Comment:
the word `fetch` indicates that we go to a DB whereas we actually extract it from the request body. how about `ExtractTokenFromCredentials `? not a blocker